lib/rules/no-unused-expressions.js

```diff
@@ -140,6 +145,9 @@ module.exports = {
             },
             FunctionExpression: alwaysTrue,
             Identifier: alwaysTrue,
+            JSXElement() {
+                return disallowJSX;
+            },
```

Comment:
Should we cover other JSX expressions that can be `ExpressionStatement#expression`, like `JSXFragment`:

```js
<></>;
```

Reply:
Yes please.

Reply:
Added.